examples/src/main/scala/org/jtech/drone/ws/AsciiServiceActor.scala

```diff
@@ -24,11 +24,13 @@ class AsciiServiceActor extends Actor {
       println(s"Unhandled message $err")
   }
 
-  def broadcast(w: WsMessage): Unit =
-    if (lastSend < System.currentTimeMillis()) {
-      lastSend += 200
+  def broadcast(w: WsMessage): Unit = {
+    val currentTime = System.currentTimeMillis
+    if (currentTime - lastTime > 32) {
```

Comment:
No, the source framerate of the webcam is unspecified and letting it loose only spams the pipeline with a lot of messages that will make the video feed appear in slow-mo (re-playing a 100fps feed at  30fps for example).

I rewrote the throttling Pim wrote and now it will drop all frames, to only take one every time more than 32ms have elapsed (1000/33 ≈ 30fps). Also, on the client side, the loop that displays frame is also running on a fixed wait time with SetInterval(..., 33).